rundeckapp/grails-app/services/rundeck/services/AuthorizationService.groovy

```diff
@@ -369,4 +378,20 @@ class AuthorizationService implements InitializingBean{
         Util.addCacheMetrics(this.class.name + ".sourceCache",registry,sourceCache)
 
     }
+
+    private void invalidateEntriesAclCache(String path) {
+        if (frameworkService.isClusterModeEnabled()) {
+            sendAndReceive(
+                    'cluster.clearAclCache',
+                    [
+                            uuidSource: frameworkService.getServerUUID(),
+                            path: path
+                    ]
+            ) { resp ->
+                log.debug("Cleaning the cache in the cluster is ${resp.clearCacheState}: ${resp.reason}")
+            }
```

Comment:
it seems it should still call `cleanCaches(path)` in cluster mode

Reply:
@gschueler cleanCaches method will be called by all active instance on cluster so I didn't think it necessary to include this call there as I would call it twice. perhaps the sender should not receive the message to execute this method via messaging service?

Reply:
core would still need it to be called